src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Theme/import.html.twig

```diff
@@ -22,24 +22,9 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
-
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 {% form_theme importThemeForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
 
-{% set layoutTitle = 'Theme & Logo'|trans({}, 'Admin.Navigation.Menu') ~ ' > ' ~ 'Theme'|trans({}, 'Admin.Design.Feature') %}
-{% set layoutHeaderToolbarBtn = {
-  'add': {
-    'href': path('admin_themes_import'),
-    'desc': 'Add new theme'|trans({}, 'Admin.Design.Feature'),
-    'icon': 'add'
-  },
-  'export': {
-    'href': path('admin_themes_export_current'),
-    'desc': 'Export current theme'|trans({}, 'Admin.Design.Feature'),
-    'icon': 'cloud_download'
-  }
-} %}
```

Comment:
This changes the layout of the page sicne it removes two buttons no?

Reply:
They are extra buttons on a subpage, import will get you to the same exact page.